.github/workflows/dependencies.yml

```diff
@@ -6,12 +6,11 @@ on:
   schedule:
     # Run at the first of every month
     - cron: "0 0 1 * *"
-  workflow_dispatch:
-    # Needed so we can run it manually
+  workflow_dispatch: # Needed so we can run it manually
 
 env:
-  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  BRANCH: cargo-update
+  AUTHOR: Update Cargo.lock Bot <bot@example.com>
```

Comment:
Why are we using a generic domain?

Reply:
No reason. I was just copying `rust-bitcoin` template since I'm familiar with that.
Any suggestions?

Reply:
NOTE: All suggestion discussed (and adopted) here I am also applying to #537.

Reply:
Is it required to include an email? If so, maybe just use a `no-reply@alpenlabs.io`.

Reply:
Done.